samples/bot-conversation/python/README.md

```diff
@@ -79,8 +79,8 @@ the Teams service needs to call into the bot.
 7) Update the `config.py` configuration for the bot to use the Microsoft App Id and App Password from the Bot Framework registration. (Note the App Password is referred to as the "client secret" in the azure portal and you can always create a new client secret anytime.)
 
 8) __*This step is specific to Teams.*__
-    - **Edit** the `manifest.json` contained in the `teamsAppManifest` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
-    - **Zip** up the contents of the `teamsAppManifest` folder to create a `manifest.zip`
+    - **Edit** the `manifest.json` contained in the `appPackage` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<AAD_APP_CLIENT_ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`)
```

Comment:
We don't need to change the folder name. We can provide the actual path in `teamsapp.yml`.

Reply:
since teamsfx cli will detect appPackage/ folder if users don't provide a manifest path, I think it is better to change it.

Reply:
let's wait for @formulahendry 's comment

Reply:
I think both ways are ok. Considering teamsfx cli, we could change the name.